src/components/VueSearchbox.jsx

```diff
@@ -86,14 +104,209 @@ const VueSearchbox = {
+      iconPosition
+    } = this.$props;
+    const { currentValue, isOpen, suggestionsList } = this.$data;
+    return (
+      <div class={className}>
+        <DownShift
+          id="searchbox-downshift"
+          handleChange={this.onSuggestionSelected}
+          handleMouseup={this.handleStateChange}
+          isOpen={isOpen}
+          scopedSlots={{
+            default: ({
+              getInputEvents,
+              getInputProps,
+
+              getItemProps,
+              getItemEvents,
+
+              isOpen,
+              highlightedIndex
+            }) => (
+              <div class={suggestionsContainer}>
+                <Input
+                  id="search-box"
+                  showIcon={showIcon}
+                  showClear={showClear}
+                  iconPosition={iconPosition}
+                  class={getClassName(innerClass, "input")}
+                  {...{
+                    on: getInputEvents({
+                      onInput: this.onInputChange,
+                      onBlur: e => {
+                        this.$emit("blur", e);
+                      },
+                      onFocus: this.handleFocus,
+                      onKeyPress: e => {
+                        this.$emit("keyPress", e);
+                      },
+                      onKeyDown: e => this.handleKeyDown(e, highlightedIndex),
+                      onKeyUp: e => {
+                        this.$emit("keyUp", e);
+                      }
+                    })
+                  }}
+                  {...{
+                    domProps: getInputProps({
+                      value:
+                        value || (currentValue === null ? "" : currentValue)
+                    })
+                  }}
+                />
+                {isOpen && suggestionsList.length ? (
+                  <ul
+                    class={`${suggestions} ${getClassName(innerClass, "list")}`}
+                  >
+                    {suggestionsList.slice(0, 10).map((item, index) => (
+                      <li
+                        {...{
+                          domProps: getItemProps({ item })
+                        }}
+                        {...{
+                          on: getItemEvents({
+                            item
+                          })
+                        }}
+                        key={`${index + 1}-${item.value}`}
+                        style={{
+                          backgroundColor: this.getBackgroundColor(
+                            highlightedIndex,
+                            index
+                          )
+                        }}
+                      >
+                        <SuggestionItem
+                          currentValue={currentValue}
+                          suggestion={item}
+                        />
+                      </li>
+                    ))}
+                  </ul>
+                ) : (
+                  <div>no suggestion</div>
```

Comment:
Should be `No Suggestions` or what we have on RS

Reply:
subsequent PR